.gitignore

```diff
@@ -0,0 +1,4 @@
+bazel-bin
```

Comment:
should be remove gitignore?

Reply:
.gitignore can be added/not added. Since building and running locally would create tons of files. So adding the .gitignore file would make it more convenience for the user to avoid large files output when executing `git status`.